api/graphql/reservations/reservation_filtersets.py

```diff
@@ -22,6 +29,12 @@ class ReservationFilterSet(django_filters.FilterSet):
     # Filter for displaying reservations which requires or had required handling.
     requested = django_filters.BooleanFilter(method="get_requested")
 
+    only_with_permission = django_filters.BooleanFilter(
+        method="get_only_with_permission"
+    )
+
+    user = django_filters.NumberFilter(method="get_user")
```

Comment:
We could use `ModelChoiceFilter` and then we don't need the get_user  method

Reply:
Oh, true! Replaced 🧹